pyhpke/__init__.py

```diff
@@ -9,7 +9,7 @@
 from .kem_key import KEMKey, KEMKeyPair
 from .kem_key_interface import KEMKeyInterface
 
-__version__ = "0.4.2"
+__version__ = "0.4.3"
```

Comment:
Could you revert the version?
I'd like to bump the version up after merging your PR.